HW4_Bobkov/protein_tools.py

```diff
@@ -0,0 +1,220 @@
+def compare(sequences: list, round_dec=3, percentages=False)->dict:
```

Comment:
1) compare не совсем понятно что делает. Лучше было бы чт нибудь в духе `calculate_similarity ` или типа того. По этом названию понятно что она делает (вычисляет) и возвращает (similarity должно быть числом).
2) round_dec тоже мб как нибудь по-дугому назвать. пресижн какой нить 

Пробелы и аннотация типов:
```suggestion
def compare(sequences: list, round_dec: int = 3, percentages: bool = False)->dict:
```